pkgs/by-name/fs/fscryptctl/package.nix

```diff
@@ -32,10 +39,10 @@ stdenv.mkDerivation rec {
       As fscryptctl is intended for advanced users, you should read the kernel
       documentation for filesystem encryption before using fscryptctl.
     '';
-    inherit (src.meta) homepage;
-    changelog = "https://github.com/google/fscryptctl/releases/tag/v${version}";
-    license = licenses.asl20;
-    platforms = platforms.linux;
-    maintainers = with maintainers; [ primeos ];
+    inherit (finalAttrs.src.meta) homepage;
```

Comment:
The url should be written out so that reviewers can easily visit it when looking at PRs.